src/main/java/org/humancellatlas/ingest/export/job/web/ExportJobSearchProcessor.java

```diff
@@ -0,0 +1,24 @@
+package org.humancellatlas.ingest.export.job.web;
+
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.linkTo;
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.methodOn;
+
+import org.humancellatlas.ingest.core.web.Links;
+import org.humancellatlas.ingest.export.job.ExportJob;
+import org.springframework.data.rest.webmvc.RepositorySearchesResource;
+import org.springframework.hateoas.ResourceProcessor;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ExportJobSearchProcessor implements ResourceProcessor<RepositorySearchesResource> {
+
+    @Override
+    public RepositorySearchesResource process(RepositorySearchesResource searchesResource) {
```

Comment:
@rolando-ebi, this isn't working on my local copy, how about for you?

Reply:
It looks like having an empty `ExportJobRepository` causes the /search to not be registered. I just added a findAll() and it worked. Will commit this change